### DISCUSSION TITLE
MFA change explained?

### DISCUSSION THREAD

**Discussion by tbee** [teslamate-org/teslamate#1161] [Q&A]
I'm the maintainer of TeslaTasks and am looking to add MFA support to it. But since the API is undocumented, I'm searching the internet to find out how. I've found that TeslaMate has added MFA support, sending the passcode along, but I cannot read the sources. What language / framework is it written in? :-D

Anyhow, is anyone inclined to explain how to send the passcode along? I suppose it is done in a HTTP header like the access token.

**Comment by ngardiner** [comment]
It's written in elixir which is a bit esoteric (or at least it is to me...) but maybe this discussion will help?

https://github.com/timdorr/tesla-api/issues/215

**Reply by tbee** [reply]
That seems like a good source, yes! Thank you! I'll try to get that working in my Java version of the API client.
